docs/core/deploying/index.md

```diff
@@ -1,95 +1,120 @@
 
-- Your app can run only if the version of .NET Core your app targets, [or a later version](../versions/selection.md#framework-dependent-apps-roll-forward), is already installed on the host system.
+If the app has platform-specific dependencies, such as a NuGet package containing platform-specific dependencies, these are copied to the publish folder along with the app.
 
-- It's possible for the .NET Core runtime and libraries to change without your knowledge in future releases. In rare cases, this may change the behavior of your app.
+A self-contained app can be created with the [`dotnet publish`](../tools/dotnet-publish.md) command.
 
-## Self-contained deployments (SCD)
+```dotnet
+dotnet publish -r <RID>
+```
 
-For a self-contained deployment, you deploy your app and any required third-party dependencies along with the version of .NET Core that you used to build the app. Creating an SCD doesn't include the [native dependencies of .NET Core](https://github.com/dotnet/core/blob/master/Documentation/prereqs.md) on various platforms, so these must be present before the app runs. For more information on version binding at runtime, see the article on [version binding in .NET Core](../versions/selection.md).
+### Advantages
 
-Starting with NET Core 2.1 SDK (version 2.1.300), .NET Core supports *patch version roll forward*. When you create a self-contained deployment, .NET Core tools automatically include the latest serviced runtime of the .NET Core version that your application targets. (The latest serviced runtime includes security patches and other bug fixes.) The serviced runtime does not have to be present on your build system; it is downloaded automatically from NuGet.org. For more information, including instructions on how to opt out of patch version roll forward, see [Self-contained deployment runtime roll forward](runtime-patch-selection.md).
+- **Control .NET Core version**\
+You control which version of .NET Core is deployed with your app. .NET Core can only be upgraded by releasing a new version of your app.
 
-FDD and SCD deployments use separate host executables, so you can sign a host executable for an SCD with your publisher signature.
+- **Platform-specific targeting**\
+Because you have to publish your app for each platform, you know where your app will run. If .NET Core introduces a new platform, users can't run your app on that platform until you release a version that is published that platform. You can test your app for compatibility problems before your users run your app on the new platform.
```

Comment:
```suggestion
Because you have to publish your app for each platform, you know where your app will run. If .NET Core introduces a new platform, users can't run your app on that platform until you release a version that is published for that platform. You can test your app for compatibility problems before your users run your app on the new platform.
```